.goreleaser.yml

```diff
@@ -77,8 +45,7 @@ archives:
     - goos: windows
```

Comment:
might clean up rest windows if not used here anymore

Reply:
sounds good, changed.